model_mr_factor: 2.0
model_delay_s: 0.15
update_freq: 40.0
max_linear_vel: 2.0
min_alt: 0.4
max_alt: 2.5

# Determines how much of the predicted feedback signal can happen in 1/fps seconds
feedback_pred_factor: 0.2
# Compensate for integerator's over-estimation of the system response
delay_compensation_factor: 0.7

safety_send_zero: true
zero_xy_hover: false
manual_override_sim: false
simulation_flag: true

pid_forward:
  p: 0.8
  i: 0.0
  d: 0.0
  i_clamp: 0.02
pid_lateral:
  p: 0.5
  i: 0.0
  d: 0.0
  i_clamp: 0.02
pid_yaw:
  p: 0.5
  i: 0.0
  d: 0.0
  i_clamp: 0.02
pid_alt:
  p: 0.5
  i: 0.0
  d: 0.0
  i_clamp: 0.02
